disperser/common/v2/blobstore/dynamo_metadata_store.go

```diff
@@ -1411,7 +1411,17 @@ func (s *BlobMetadataStore) GetSignedBatch(ctx context.Context, batchHeaderHash
 	}
 
 	if attestation == nil {
-		return nil, nil, fmt.Errorf("%w: attestation not found for hash %x", common.ErrMetadataNotFound, batchHeaderHash)
+		// if no attestation is found, return an empty attestation
```

Comment:
The merge process with my signature gathering PR must have gone wrong, pretty sure this shouldn't be changing

Reply:
I've reverted all changes to this file.